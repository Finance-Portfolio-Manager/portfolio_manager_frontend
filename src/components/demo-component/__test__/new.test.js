import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import News from './../new';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';

import { shallow, ShallowWrapper, mount } from "enzyme";

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
const wrapper = mount(<News></News>);

test("matches snapshot", ()=>{
    const tree = renderer.create(<News></News>).toJSON();
    expect(tree).toMatchSnapshot();
})