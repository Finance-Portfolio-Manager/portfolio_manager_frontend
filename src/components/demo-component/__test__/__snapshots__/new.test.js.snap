// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container-fluid "
>
  <div
    class="carousel slide"
    data-bs-ride="carousel"
    id="carouselExampleControls"
  >
    <div
      className="carousel-inner"
    />
    <button
      className="carousel-control-prev"
      data-bs-slide="prev"
      data-bs-target="#carouselExampleControls"
      type="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="visually-hidden"
      >
        Previous
      </span>
    </button>
    <button
      className="carousel-control-next"
      data-bs-slide="next"
      data-bs-target="#carouselExampleControls"
      type="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="visually-hidden"
      >
        Next
      </span>
    </button>
  </div>
</div>
`;
